### ISSUE TITLE
Sequences of digits are being interpreted as numbers on image files

### ISSUE THREAD

**Issue by poveden** [axa-group/Parsr#129]
**Summary**

When an image (e.g. PNG, JPG) contains sequences of digits, they are being treated as a number instead of a sequence of characters.

For example, the sequence `0002` is parsed as `2`.

This is troublesome when all digits are required to be present to have a meaning (e.g. bank account numbers).

This seems to affect, at least, text, markdown, and JSON outputs.

**Steps To Reproduce**

1. Prepare an image that contains the number sequence `0002`
2. Post to the `/api/document` endpoint the image along with the [example configuration file](https://github.com/axa-group/Parsr/blob/develop/docs/configuration-file.md#5-exempli-gratia)
3. Retrieve the outputs from `/api/json/{id}`, `/api/text/{id}` and `/api/markdown/{id}`

**Expected behavior**

- The JSON output should contain a `word` element with `"content": "0002"`
- The text and markdown outputs should contain the string `0002`

**Actual behavior**

- The JSON output contains a `word` element with `"content": "2"`
- The text and markdown outputs should contain the string `2`

**Environment**
- Docker Desktop for Windows 2.1.0.3 (38240), engine 19.03.2
- Docker image `axarev/parsr`, ID `74a4e7320413`

**Additional context**
Test images can be provided on request.

**Comment by aarohijohal** [comment]
Well caught @poveden !
I tried with this particular TSV output from tesseract:
```
level   page_num        block_num       par_num line_num        word_num        left    top     width   height  conf    text
1       1       0       0       0       0       0       0       640     480     -1
2       1       1       0       0       0       120     170     187     278     -1
3       1       1       1       0       0       120     170     187     278     -1
4       1       1       1       1       0       120     170     70      47      -1
5       1       1       1       1       1       120     170     70      47      95      02
4       1       1       1       2       0       120     247     109     47      -1
5       1       1       1       2       1       120     247     109     47      96      002
4       1       1       1       3       0       120     324     148     47      -1
5       1       1       1       3       1       120     324     148     47      96      0002
4       1       1       1       4       0       120     401     187     47      -1
5       1       1       1       4       1       120     401     187     47      95      00002
```

and the library [TSV](https://www.npmjs.com/package/tsv) just reads all of them as 2 instead of (0..)2:
```
2  
2 2 2

```

I don't see a way to tweak it to act differently.
Maybe we can use `sv` or `tsv-to-database`; but I have to admit, `tsv-to-database` looks like overkill for a simple task.

**Comment by poveden** [comment]
> and the library [TSV](https://www.npmjs.com/package/tsv) just reads all of them as 2 instead of (0..)2:

Why don't we just implement this functionality directly? The TSV library is quite simple, and [the culprit code](https://github.com/ricardobeat/TSV/blob/ac61fe2c0f7744100580d90e8b8c9781b4114f98/index.js#L28-L29) can easily be fixed.

If you want, I can push a PR for you.

**Comment by aarohijohal** [comment]
@poveden Please feel free to do so. Thanks!

**Comment by aarohijohal** [comment]
Fixed by Commit ec9048c1b177efc600e64a6a5d315d9c22773f81
